src/server/handlers/launchpad.js

```diff
@@ -301,3 +301,57 @@ export const completeSnapAuthorization = async (req, res) => {
     });
   });
 };
+
+export const getSnapBuilds = (req, res) => {
+  const snap_link = req.query.snap_link;
+
+  if (snap_link) {
+    return getLaunchpad().get(snap_link).then(snap => {
+      // TODO: bartaz: currently just limiting to 10 recent builds
+      // we will need to process it properly later for paging/loading more
+      return getLaunchpad().get(snap.builds_collection_link, { start: 0, size: 10 })
+        .then(builds => {
+          return res.status(200).send({
+            status: 'success',
+            payload: {
+              code: 'snap-builds-found',
+              message: builds.entries
```

Comment:
@katiefenn @cjwatson I wanted to follow the response convention we use here (to have `code` and `message` in payload), but I'm not sure if array of builds actually should go to message here? Or should it be rather separate field?